src/styles/app/18-filetree-body.css

```
@@ -369,6 +369,15 @@
 	box-shadow: var(--pill-shadow);
 }
 
+.fileTreePinIcon {
+	flex-shrink: 0;
+	color: #e8b830;
+}
+
+.fileTreePinIcon path {
+	fill: currentColor;
```

Stylelint (17.13.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)